src/docfx/build/legacy/files/LegacyOutput.cs

```diff
@@ -2,33 +2,40 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Collections.Generic;
-using System.IO;
 using System.Threading.Tasks;
 
 namespace Microsoft.Docs.Build
 {
     internal static class LegacyOutput
     {
-        public static void Convert(Docset docset, Context context, List<(LegacyManifestItem manifestItem, Document document)> files, TableOfContentsMap tocMap)
+        public static async Task Convert(Docset docset, Context context, List<(LegacyManifestItem manifestItem, Document document)> files, TableOfContentsMap tocMap)
```

Comment:
I would like always use (finalized)List instead of IEnumerable because you never know if the IEnumerable is finalized/executed or not.

---
In reply to: [208101451](https://github.com/dotnet/docfx/pull/3161#discussion_r208101451) [](ancestors = 208101451)